client/app/components/Input/InputSwitch.jsx

```diff
@@ -37,7 +37,7 @@ export function InputSwitch({
     setKey(Utils.randomString());
   };
 
-  const onKeyPress = (e: SyntheticKeyboardEvent<HTMLInputElement>) => {
```

Comment:
Small update: replaces [onKeyPress](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onkeypress) with `onKeyDown`, should still work without issue